build/common/nginx-default.conf.template

```diff
@@ -29,9 +29,9 @@ server {
         proxy_http_version 1.1;
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "upgrade";
-        proxy_set_header X-Frappe-Site-Name $http_host;
+        proxy_set_header X-Frappe-Site-Name $host;
         proxy_set_header Origin $scheme://$http_host;
-        proxy_set_header Host $host;
+        proxy_set_header Host $http_host;
```

Comment:
Is this change necessary?

https://github.com/frappe/bench/blob/develop/bench/config/templates/nginx.conf#L91

Here it is set to $host instead of $http_host?

Reply:
This [github issue](https://github.com/nginx-proxy/nginx-proxy/issues/763) on official nginx repo could answer your question as to why use $http_host to set the Host header.